src/test/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/RecepMessageDateTimeTest.java

```diff
@@ -0,0 +1,46 @@
+package uk.nhs.digital.nhsconnect.lab.results.model.edifact;
+
+import org.junit.jupiter.api.Test;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.message.EdifactValidationException;
+
+import java.time.Instant;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class RecepMessageDateTimeTest {
```

Comment:
Can we create a test covering a null timestamp?

Reply:
Done